docs/Command-Line-Interface.md

```diff
@@ -182,3 +209,335 @@ Or, if no header is provided, the default hostname and port (set in the env file
     <td>Remove the TTL (time to live) setting from a file.</td>
   </tr>
 </table>
+
+# Example Use Cases
+
+## cat
+The `cat` command will print the entire contents of a file in Tachyon to the console. This can be
```

Comment:
NIT: most other examples uses present tense as the first sentence (e.g., fileinfo and below), while `cat`,  `copyFromLocal` and a few uses future tense. Shall we keep consistency across this section?